src/plugins/optimize-graph-plugin.js

```diff
@@ -176,15 +176,31 @@ class ChunkGraph {
 				let rep = typeof newUrl === 'function' ? newUrl(url, fn, quote) : newUrl;
 				if (rep === false) return 'null';
 				if (typeof rep === 'string') {
-					if (publicPath) rep = posix.join(publicPath, rep);
-					rep = JSON.stringify(rep);
+					rep = publicPath ? toImport(publicPath, rep, true) : JSON.stringify(rep);
```

Comment:
As per [other comment](https://github.com/preactjs/wmr/pull/213/files#r535578173):

```suggestion
					rep = JSON.stringify(publicPath ? toImport(publicPath, rep) : rep);
```